### DISCUSSION TITLE
Cuegen - A tool for building Kubernetes resources with CUE

### DISCUSSION THREAD

**Discussion by nxcc** [cue-lang/cue#2226] [Show and tell]
Building Kubernetes resources with CUE is fun, but coming from kustomize, it lacks an comfortable way to load external resources, such as loading configuration files and scripts into configmaps. It is also not possible to include (on disk) encrypted data such as certificates and passwords. For all this, and also the versioned import of external CUE resources, cuegen provides a possible solution. 

Cuegen is avaliable at https://github.com/noris-network/cuegen

**Comment by Lyoness** [comment]
This is great! 🎉
